console.js

```diff
@@ -1,13 +1,13 @@
 // vim: tw=120 softtabstop=4 shiftwidth=4
 
-let fs = require('fs')
-let tracer = require('tracer');
+const fs = require('fs')
+const tracer = require('tracer');
 
-let config = require('./config');
+const config = require('./config');
 
-let console_fmt = "{{timestamp}} {{title}} {{message}}";
-if (config.DEBUG)
-    console_fmt = "{{timestamp}} {{title}} {{file}}:{{line}}{{space}} {{message}}";
+const console_fmt = "{{timestamp}} {{title}} " +
+                    (config.DEBUG ? "{{file}}:{{line}}{{space}} {{message}}"
+                                  : "{{message}}");
```

Comment:
Instead of doing `config.DEBUG ? "A B C" : "C"` we could do `(config.DEBUG ? "A B" : "") + C`

```
const console_fmt = `{{timestamp}} {{title}} ${(config.DEBUG ? "{{file}}:{{line}}{{space}}" : ""}{{message}}`;
```

Reply:
indeed, i was so focused in the first fix didnt see we could also do that :)